packages/components/src/organisms/ImageGallery/ImageZoom.tsx

```diff
@@ -1,12 +1,14 @@
 import type { PropsWithChildren } from 'react'
+import React from 'react'
 
-interface ImageZoomProps {
+// TODO: adds zoom feature to the selected image
+export interface ImageZoomProps {
   helpMessage?: string
   zoomFactor?: number
 }
 
 const ImageZoom = ({ children }: PropsWithChildren<ImageZoomProps>) => {
```

Comment:
Should we keep this component? Maybe we can remove it, and add it later when the feature is available? 🤔

Reply:
I like the idea of leaving the component to reinforce the `to do` for future tasks.